buildSrc/src/main/groovy/org/gradle/testretry/build/GradleVersionData.groovy

```diff
@@ -35,4 +42,24 @@ class GradleVersionData {
             .collect { it.key.toString() }
     }
 
+    static int major(GradleVersion gradle) {
```

Comment:
The `GradleVersion` class unfortunately doesn't provide methods for accessing the major/minor number of the Gradle version.